lib/package.json

```
@@ -5,8 +5,8 @@
   "description": "An extensible form builder based on React JSON Schema Form.",
   "scripts": {
     "build": "tsc && vite build",
-    "build:check": "if [ ! -d dist ] || [ $(find src -type f -newer dist) ] ; then yarn build ; fi",
-    "build:watch": "yarn build --watch",
+    "build:check": "if [ ! -d dist ] || [[ -n $(find src -type f -newer dist) ]] ; then yarn build ; fi",
```

bin/sh complains about `[[`on my side
```suggestion
    "build:check": "if [ ! -d dist ] || [ $(find src -type f -newer dist | head -1) ] ; then yarn build ; fi",
```

---

🙏